### ISSUE TITLE
Editorial problems in focus appearance 

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag#2570]
On the current editors draft: https://w3c.github.io/wcag/guidelines/22/#focus-appearance

1. The second option uses the phrase "meets all the following", the first option, which seems to work the same does not have this wording. This feels unnecessarily inconsistent.

2. The notes have the word "NOTE" in them twice:

> NOTE
> Note: Examples of sub-components...

> NOTE
> Note: Contrast calculations can ...

3. The "minimum bounding box" definition has a typo:
> such has hypertext links

Should be "such as" not "such has".

**Comment by alastc** [comment]
There are PRs in survey for this.

**Comment by alastc** [comment]
All done.
